composer.json

```diff
@@ -16,20 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
-        "mybuilder/cronos": "~3.0",
-        "doctrine/annotations": "1.*",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
-        "symfony/property-access": "^4.4 || ^5.0 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0"
+        "php": ">=8.0.2",
+        "mybuilder/cronos": "^3.0 || ^4.0",
+        "doctrine/annotations": "^1.14 || ^2.0",
+        "symfony/console": "^4.4.40 || ^5.4 || ^6.0",
```

Comment:
Does it need to be as specific as `4.4.40`? If we allow any version of doctrine 1 or 2 will composer not work out which one is compatible?

Reply:
> There is a problem with symfony 4 and doctrine/annotations v2. And as it lost LTS i guess it will not be implemented anymore. If we want to support symfony 4, i guess we can not use doctrine/annotations v2. "doctrine/annotations": "^1.14 || ^2.0",
> 
> It will work on symfony 4 with 1.14 (1.x) but i can not set this logic in composer.
> 
> Because of this, i suggest to drop the support of symfony 4. Or drop the support of "doctrine/annotations" v2. But this will propably block future doctrine updates for the users.
> 
> WDYT?

Which part of Symfony 4 doesn't work with doctrine annotations? Do you have an example to the problem you were seeing? Ideally would be good to support both

Reply:
We do not need `4.4.40` this was more for my local testing and errors deprecations with. But we need `4.4.20` for php 8 and deprecation free.

The problem is here in this testrun (see below).
I could not figure out how to use symfony 4.4. with doctrine/annotations v2 ;(
It can propably be solved with the same fix as symfony did to 5.4 and 6.2. I guess many bundles have this problem. Maybe i oversee something, and it can work together. But i do not know how.

https://github.com/Chris53897/cronos-bundle/actions/runs/3837872124/jobs/6533672205

` 1) MyBuilder\Bundle\CronosBundle\Tests\Command\DumpCommandTest::test_dump_should_be_as_expected with data set #0 ('Server all\nFound 3 lines\nWe...v=test', array('test'))
Error: Call to undefined method Doctrine\Common\Annotations\AnnotationRegistry::registerLoader()

/home/runner/work/cronos-bundle/cronos-bundle/var/cache/test/ContainerS13molL/getMybuilder_CronosBundle_AnnotationCronExporterService.php:28
/home/runner/work/cronos-bundle/cronos-bundle/var/cache/test/ContainerS13molL/appMyBuilder_Bundle_CronosBundle_Tests_Fixtures_app_AppKernelTestDebugContainer.php:126
/home/runner/work/cronos-bundle/cronos-bundle/vendor/symfony/dependency-injection/Container.php:237
/home/runner/work/cronos-bundle/cronos-bundle/vendor/symfony/dependency-injection/Container.php:219
/home/runner/work/cronos-bundle/cronos-bundle/Command/CommandBase.php:63
/home/runner/work/cronos-bundle/cronos-bundle/Command/CommandBase.php:33
/home/runner/work/cronos-bundle/cronos-bundle/Command/DumpCommand.php:30
/home/runner/work/cronos-bundle/cronos-bundle/vendor/symfony/console/Command/Command.php:255
/home/runner/work/cronos-bundle/cronos-bundle/vendor/symfony/console/Tester/CommandTester.php:76
/home/runner/work/cronos-bundle/cronos-bundle/Tests/Command/DumpCommandTest.php:44`

Reply:
`If we allow any version of doctrine 1 or 2 will composer not work out which one is compatible?`
I do not why this is not restricting this. Maybe there is just not a restriction in composer in symfony.

Reply:
Ah ok, I can see that in FrameworkExtension it calls this which as you say is deprecated in V1 and then removed in V2. I assumed Framework bundle had a dependancy on `doctrine/annotations` but it does not so as you say it will get the latest version :( 
Think we'd need the fix here in 4.4 which won't happen as it's no longer supported: https://github.com/derrabus/symfony/blob/650f7bd03e1ae36412b2099976831042472ae91c/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1634-L1641 
🤔 I think for now let's leave it as you've done in this PR for V1 and I can have a think about how we resolve this

Reply:
Thanks for investigating. That looks like the correct change.

> I think for now let's leave it as you've done in this PR for V1 and I can have a think about how we resolve this

I agree. If you have time you can send a PR to the 4.4 branch. Maybe they will accept it. It looks like an easy PR.
But i will go on vaccation for a week.